dev/before_install

```diff
@@ -12,3 +12,5 @@ elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
 	brew upgrade python
 	pip3 install pep8 flake8
 fi
+
+sudo ./dev/install-universal_ctags.sh
```

Comment:
On travis it's enough to have it in path? No need to set the `SYSTEM_CTAGS_PROPERTY` for indexer?
I mean if the location of the installation does or doesn't matter here.

Reply:
It seems that in Travis environment it is sufficient to have the ctags binary in `PATH`. The test run reports
```
[WARNING] Tests run: 727, Failures: 0, Errors: 0, Skipped: 6
```
which is in line what is reported for `master`:
```

```